base_kanban_stage_state/models/base_kanban_stage.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Specialty Medical Drugstore
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+_STATE = [
```

Comment:
On second thought, this should actually be an `api.model` method in `base.kanban.stage` - `_get_states()`. Allows for easier inheritance